### ISSUE TITLE
Benchmark build failure

### ISSUE THREAD

**Issue by mihaimaruseac** [Bodigrim/bit-stream#4]
On stackage, log below

```haskell
Building benchmark 'bench' for bit-stream-0.1.0.1..
[1 of 1] Compiling Main             ( bench/Bench.hs, dist/build/bench/bench-tmp/Main.o )

bench/Bench.hs:26:23: error:
    Variable not in scope: toIdx :: Word -> Word
   |
26 |   , doBench "toIdx" $ toIdx
   |                       ^^^^^
```

**Comment by Bodigrim** [comment]
Thanks. Should be fixed for now; released as 0.1.0.2.

**Comment by mihaimaruseac** [comment]
Thanks. It works now.
